### PR TITLE
[Estuary][PVR] Guide window: Add possibility to switch between horizontal and vertical epg grid view.

### PR DESCRIPTION
**Pull request by ksooo**
This PR enhances Estuary's PVR Guide window with the feature to be able to choose between two epg grid views - 'vertical' (already present before this PR) and 'horizontal' (new).

screenshot000
screenshot001
screenshot002

The changes have been runtime tested on macOS, latest Kodi master.

@phil65 @ronie might be interested in taking a look at the skin changes.

### CODE REVIEW

File: addons/skin.estuary/xml/Includes_MediaMenu.xml [hunk 1]
```diff
@@ -123,6 +122,14 @@
 					<include>MediaMenuLabelCommon</include>
 					<label>$LOCALIZE[14018]</label>
 				</control>
+				<control type="button" id="6051">
+					<include>MediaMenuItemsCommon</include>
+					<label>$LOCALIZE[31023]</label>
+					<label2>[B]$INFO[Container.Viewmode][/B]</label2>
+					<visible>Integer.IsGreater(Container.ViewCount,1)</visible>
```
Comment (phil65) [inline addons/skin.estuary/xml/Includes_MediaMenu.xml]: this condition is not needed (cointainer.viewcount is always 2 I guess)

Comment (ksooo) [inline addons/skin.estuary/xml/Includes_MediaMenu.xml]: Yes, you are right, at least as long as we have the other 'visible' condition in place (which I actually added later). I will remove this one.

File: addons/skin.estuary/xml/Includes_PVR.xml [hunk 2]
```diff
@@ -157,4 +157,165 @@
...
+						<height>60</height>
+						<font>font32_title</font>
+						<label>$INFO[ListItem.ChannelNumberLabel]</label>
+						<align>center</align>
+						<aligny>center</aligny>
+					</control>
+					<control type="label" id="1">
+						<left>68</left>
+						<top>-2</top>
+						<width>$PARAM[channel_label_width]</width>
```
Comment (phil65) [inline addons/skin.estuary/xml/Includes_PVR.xml]: Is this width needed? I thaught the purpose of id=1 in these layouts is that the width is set by core?

**Comment by phil65** [comment]
2 comments, but looks good in general.

File: addons/skin.estuary/xml/Includes_PVR.xml (hunk 2 above)
Comment (ksooo) [inline addons/skin.estuary/xml/Includes_PVR.xml]: Did not know this. Runtime-tested. Works like a charm without the 'width' definitions.

**Comment by ksooo** [comment]
@phil65 your suggested changes are now in. Thanks for the review.

**Comment by ksooo** [comment]
jenkins build this please

**Pull request merged**